### ISSUE TITLE
[New build - DEFAULT] 20w48c: Hotfixes

### ISSUE THREAD

**Issue by tsunamayo** [tsunamayo/Starship-EVO#3304]
Some more fixes.

Bugfixes:
#3293 Game not saving on quit.
#3298 Game FPS drop on warp in some instance.
#3298 Teleport moves player to incorrect position.
#3299 Copy-Paste can copy the undeletable children base brick of a mechanism.

**Comment by tsunamayo** [comment]
build is up! Thanks all!

**Comment by MagicalAbyss** [comment]
Hi, about the undeletable brick, are you able to change it to be able to delete on the existing blueprint cause by the bug? Thanks for quick fix.

**Comment by tsunamayo** [comment]
nope unfortunately. Was it on a big project of yours?

**Comment by MagicalAbyss** [comment]
> nope unfortunately. Was it on a big project of yours?

Yes, but fortunately found a way to delete them, trying to reproduce this "bug".
